lang_tests/szext.c

```diff
@@ -0,0 +1,21 @@
+// Compiler: 
+// Runtime:
+//    stdout: 6
+//            1
+//            0
+void main() {
+    int i1 = 6;
+    long i2 = (long) i1;
+    printf("%i", i2);
+
+    int i3 = 2;
+    int b1 = i3 == i3;
```

Comment:
Is it correct that this compares `i3` with itself?

Reply:
Yes I just wanted a true comparison.

Reply:
Ok.